docs/network_firewall.rst

```diff
@@ -28,14 +28,14 @@ If you are new to pfSense or firewall management in general, we
 recommend the following resources:
 
 -  `Official pfSense
-   Wiki <https://doc.pfsense.org/index.php/Main_Page>`__
+   Wiki <https://docs.netgate.com/pfsense/en/latest/index.html>`__
 -  `pfSense: The Definitive
    Guide <https://www.amazon.com/pfSense-Definitive-Christopher-M-Buechler/dp/0979034280>`__
 
    -  *Note:* This guide is now slightly out of date, although we found
       it to be a useful reference in the past. To get the latest version of
       this book, you need to become a `pfSense Gold
-      Member <https://www.pfsense.org/our-services/gold-membership.html>`__.
+      Member <https://www.pfsense.org/our-services/#gold-membership>`__.
```

Comment:
Note that Gold Membership doesn't seem to be offered by pfSense anymore, and `make linkcheck` reports that the `#gold-membership` anchor is not found.

I found references to a "version 2.1" published in 2013, but I didn't find any in the pfSense docs and I don't know how it can be obtained anymore.

Reply:
I'm inclined to remove this link altogether; it was flagged as "slightly out of date" more than 5 years ago (https://github.com/freedomofpress/securedrop/commit/8be932c1932d22fc75e684573c67329271e1a84f). @zenmonkeykstop or others; any thoughts on other pfSense resources we may want to cross-reference here?

Reply:
https://docs.netgate.com/pfsense/en/latest/index.html is a good resource, which is linked elsewhere apparently - I'd agree with just removing this one along with the obsolete info about memberships.